deployment/copilot/devDeploy.sh

```diff
@@ -186,8 +184,7 @@ else
   echo "No auth header options detected for Replayer, defaulting to not specifying an explicit auth header"
 fi
 replay_command_base="/bin/sh -c \"/runJavaWithClasspath.sh org.opensearch.migrations.replay.TrafficReplayer https://${MIGRATION_DOMAIN_ENDPOINT}:443 --insecure --kafka-traffic-brokers ${MIGRATION_KAFKA_BROKER_ENDPOINTS} --kafka-traffic-topic logging-traffic-topic --kafka-traffic-group-id default-logging-group --kafka-traffic-enable-msk-auth "
-replay_command_end=" | nc traffic-comparator 9220\""
-replay_command="${replay_command_base}${auth_header_args}${replay_command_end}"
+replay_command="${replay_command_base}${auth_header_args}
```

Comment:
Why is there only one quotation here? I think we will need to add an additional quotation mark at the end of the command as the replay_command_base leaves one open